option optimize_for = LITE_RUNTIME;
import "U64Type.proto";
message RebelShareFlagNotify
{
 required U64Type serial_id = 1;
}
